HardwareConnection.java

```diff
@@ -43,6 +75,37 @@ public HardwareConnection(){
     public void init(HardwareMap ahwMap) {
         // Save reference to Hardware map
         hwMap = ahwMap;
+
+        // Define and Initialize Motors
+        motor_left_back = hwMap.get(DcMotor.class, "motor_left_back");
+        motor_left_front = hwMap.get(DcMotor.class, "motor_left_front");
+        motor_right_back = hwMap.get(DcMotor.class, "motor_right_back");
+        motor_right_front = hwMap.get(DcMotor.class, "motor_right_front");
+        //motor_middle    = hwMap.get(DcMotor.class, "motor_middle");
+        upper_grip = hwMap.get(Servo.class, "upper_grip");
+        lower_grip = hwMap.get(Servo.class, "lower_grip");
+
+
+        motor_left_back.setDirection(DcMotor.Direction.FORWARD); // Set to REVERSE if using AndyMark motors
+        motor_left_front.setDirection(DcMotor.Direction.REVERSE);// Set to FORWARD if using AndyMark motors(we have placed it in reverse)
+        motor_right_back.setDirection(DcMotor.Direction.FORWARD); // Set to REVERSE if using AndyMark motors
+        motor_left_front.setDirection(DcMotor.Direction.FORWARD); // Set to REVERSE if using AndyMark motors
+        //motor_middle.setDirection(DcMotor.Direction.FORWARD); // Set to REVERSE if using AndyMark motors
+
+        // Set all motors to zero power
+        motor_left_back.setPower(0);
```

Comment:
Create a function to set the power for all drive motors.
call it something like setDriveMotorsPower